blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -41,79 +50,176 @@ industry. We are committed to making a positive impact on the community
 
-coreboot is an open-source firmware that provides a lightweight, secure,
+[coreboot](https://3mdeb.com/training/#coreboot-for-embedded-linux-developers)
+is an open-source firmware that provides a lightweight, secure,
 and fast boot experience for PCs, laptops, servers, and embedded devices.
+The changes made by our engineers can be divided into three main groups.
+
+#### Adding support for new platforms and maintaining existing ones
+
+Recently, our engineers have focused on expanding coreboot support for the
+latest platforms, it is worth mentioning that Karol Zmyslowski has added
+[support for Jasper Lake](https://review.coreboot.org/c/coreboot/+/73934).
+
+Michał Kopeć and Michał Żygowski had a major impact on Alder Lake support in
+coreboot. Michał Kopeć has added [support for Alder Lake](https://review.coreboot.org/c/coreboot/+/63374)
+and Michał Żygowski in the [set of patches](https://review.coreboot.org/c/coreboot/+/68449/11)
+has added improvements huge improvements. Also, Michał Żygowski made a significant
+contribution to [Tiger Lake support](https://review.coreboot.org/c/coreboot/+/56171).
+
+In the previous year, our team also added
+[support for MSI](https://review.coreboot.org/c/coreboot/+/68078) PRO Z690-A
+WIFI and MSI PRO Z690-A in DDR4 and DDR5 versions.
+
+Krystian Hebel, Sergii Dmytruk, and Yaroslav Kurlaev have made major contributions
+to developing and maintaining support for Power platforms. Their changes include
+[emulation improvements](https://review.coreboot.org/c/coreboot/+/67061),
+and fixing ongoing issues related to these platforms.
+
+#### Platform security (TPM, TXT)
+
+Our team also worked on [TPM](https://3mdeb.com/shop/modules/tpm-2-0/) and
+[TXT](https://blog.3mdeb.com/2022/2022-03-17-optiplex-txt/) solutions that
+could support the security of coreboot-based platforms. Sergii Dmytruk added
+quite a few improvements related to
+[TPM support](https://review.coreboot.org/c/coreboot/+/68748),
+while Michal Zygowski added some
+[TXT improvements](https://review.coreboot.org/c/coreboot/+/59519).
+
+#### coreboot documentation improvements
+
+The work of our engineers was not closed only to changes in the code.
+It is equally important to make the code readable, clear, and understandable
+to everyone. Hence, it is worth highlighting our efforts to keep the coreboot
+documentation up to date. In this matter, the main contributors were
+[Michał Żygowski](https://review.coreboot.org/c/coreboot/+/65702) and
+[Sergii Dmytruk](https://review.coreboot.org/c/coreboot/+/68752)
+
 If you are interested in improving the security of your devices while reducing
 your dependence on proprietary firmware, you could benefit from using coreboot.
 We can also help you reduce time-to-market by simplifying the firmware
-development process.
+development process. As well as if you want your firmware to be user-friendly
+and well-documented user-friendly and well-documented then coreboot is for you.
```

Comment:
Instead of this paragraph, I would try to pitch Dasharo, especially since most of the above changes were part of Dasharo's product development.

Reply:
Pitch added https://github.com/3mdeb/news-and-ideas/pull/519/commits/6232e10165348e655ed155ab352d394464008f15#diff-5dff579917e1df023c3d68f093f637f7fb497ee2f22650cc786ac30a195a7eceR85